component/site/libraries/commonfunctions.php

```diff
@@ -435,29 +437,38 @@ public static function sendAdminMail( $adminName, $adminEmail, $subject='', $tit
                     $jevadminuser = new  JUser($params->get("jevadmin",62));
                     if ($jevadminuser->email != $adminEmail){
                             $mail->addRecipient($jevadminuser->email);
-                    }                    
+                    } else {
+                    	$mail->addRecipient($adminEmail);
+                    }
                 }
 
 		/**
 		 *
 		 * TODO - pass message through layout template processor
 		 *
 		 */
-		
+
 		$mail->setSubject($subject);
 		$mail->setBody($messagetemplate);
-                
+
 		if ($event){
 			$dispatcher     = JEventDispatcher::getInstance();
 			JPluginHelper::importPlugin("jevents");
 			$res = $dispatcher->trigger( 'onSendAdminMail' , array(&$mail, $event));
 		}
-                
+
 		if ($cc!=""){
 			$mail->addCC($cc);
 		}
 		$mail->IsHTML(true);
-		$mail->send();
+
+		//Check not emailing the same user who is editing:
+		$user = JFactory::getUser();
+
+		if ($user->email !== $adminEmail){
```

Comment:
Ok so what about then adding it at line 438 with:

if ($user->email !== $adminEmail){ return; } Thus only apply to the Admin email?